Classes/COSTouchVisualizer.xcodeproj/project.pbxproj

```diff
@@ -209,10 +211,14 @@
 				GCC_NO_COMMON_BLOCKS = YES;
 				INFOPLIST_FILE = Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				IPHONEOS_DEPLOYMENT_TARGET = 12.4;
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/Frameworks",
+					"@loader_path/Frameworks",
+				);
 				MTL_ENABLE_DEBUG_INFO = YES;
-				PRODUCT_BUNDLE_IDENTIFIER = com.conopsys.TouchVisualizer;
+				PRODUCT_BUNDLE_IDENTIFIER = com.multitracks.TouchVisualizer;
```

Comment:
@multitracks Should we be updating the bundle identifier of these forked repos to reference multitracks.com?

Reply:
Sounds good 👍